[2025-05-08T14:41:28.012-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T14:41:28.017-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T19:40:40+00:00 [queued]>
[2025-05-08T14:41:28.021-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T19:40:40+00:00 [queued]>
[2025-05-08T14:41:28.021-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-08T14:41:28.036-0500] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): modoOperacion> on 2025-05-08 19:40:40+00:00
[2025-05-08T14:41:28.042-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'request_operation', 'modoOperacion', 'scheduled__2025-05-08T19:40:40+00:00', '--job-id', '2194', '--raw', '--subdir', 'DAGS_FOLDER/operations.py', '--cfg-path', '/tmp/tmpur_hyr7x']
[2025-05-08T14:41:28.044-0500] {standard_task_runner.py:105} INFO - Job 2194: Subtask modoOperacion
[2025-05-08T14:41:28.045-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=107548) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-08T14:41:28.045-0500] {standard_task_runner.py:72} INFO - Started process 107579 to run task
[2025-05-08T14:41:28.070-0500] {task_command.py:467} INFO - Running <TaskInstance: request_operation.modoOperacion scheduled__2025-05-08T19:40:40+00:00 [running]> on host DESKTOP-0R7MK50.
[2025-05-08T14:41:28.124-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='request_operation' AIRFLOW_CTX_TASK_ID='modoOperacion' AIRFLOW_CTX_EXECUTION_DATE='2025-05-08T19:40:40+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-08T19:40:40+00:00'
[2025-05-08T14:41:28.125-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T14:41:29.331-0500] {logging_mixin.py:190} WARNING - /home/ares/anaconda3/envs/iqoptions_env/lib/python3.12/site-packages/urllib3/connectionpool.py:1097 InsecureRequestWarning: Unverified HTTPS request is being made to host 'auth.iqoption.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
[2025-05-08T14:41:31.119-0500] {python.py:240} INFO - Done. Returned value was: Modo PRACTICE. Balance actual: 9913.1
[2025-05-08T14:41:31.139-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T14:41:31.139-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=request_operation, task_id=modoOperacion, run_id=scheduled__2025-05-08T19:40:40+00:00, execution_date=20250508T194040, start_date=20250508T194128, end_date=20250508T194131
[2025-05-08T14:41:31.186-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T14:41:31.196-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T14:41:31.197-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
